kernel/loxilb_libdp.c

```diff
@@ -1320,7 +1320,7 @@ llb_clear_map_stats(int tid, __u32 idx)
   t = &xh->maps[tid];
   if (t->has_pb && t->pb_xtid <= 0) {
     /* FIXME : Handle non-pcpu */
-    if (idx >= 0) {
+    if ((__s32)idx >= 0) {
```

Comment:
Typecasting to __s32 means anything greater than 2(pow)16 will fail this check which is not exactly desired. Suggest to revert this change in this file for now.

Reply:
This check is removed completely as it was unnecessary. Thanks @cybwan